taxonomies/categories.txt

```diff
@@ -1822,6 +1846,63 @@ ro:Băuturi spirtoase
 zh:硬酒
 wikidata:en:Q2832208
 
+<en:Alcoholic beverages
+en:German liqueur
+
+<en:German liqueur
+de:Bayerischer Blutwurz
+protected_name_file_number:en: PGI-DE-02581
+protected_name_type:en: gi
```

Comment:
gi? not pgi?

Reply:
![image](https://user-images.githubusercontent.com/1689815/106299545-936bf480-6255-11eb-953f-2da5c6570701.png)
It's not clear: Sui Generis
![image](https://user-images.githubusercontent.com/1689815/106299596-a383d400-6255-11eb-9fb6-577410bcf76c.png)

Reply:
https://www.origin-gi.com/i-gi-origin-worldwide-gi-compilation-uk.html?filter=Chiemseer%20Klosterlik%C3%B6r&cc=p